website/pages/docs/concepts/writing-styles.md

```diff
@@ -218,11 +218,94 @@ const styles = css({
 })
 
 const Card = ({ className, ...props }) => {
-  const rootClassName = cx(styles, className)
+  const rootClassName = cx('group', styles, className)
   return <div className={rootClassName} {...props} />
 }
 ```
 
+### Overriding
+
+Passing multiple styles to the `css` function will merge the styles together. This allows you to override styles in a predictable way.
+
+```jsx
+import { css } from '../styled-system/css'
+
+const result = css({ mx: '3', paddingTop: '4' }, { mx: '10', pt: '6' })
+//    ^? result = "mx_10 pt_6"
+```
+
+If you intend for your components to have overridable styles, rather than passing
+the result of the `css` function as props, you should pass the style objects using the `raw` functions and just call the
+`css` function in the component itself.
+
+Example:
+
+```tsx title="src/components/Button.tsx"
+'use client'
```

Comment:
```suggestion
```

Reply:
I left this one on purpose cause [the minimal reproduction that lead to this change](https://github.com/Jacobjepsen27/panda-css-rsc-issue/blob/main/src/components/Button/Button.tsx#L1) was using RSC (and `cx` wouldn't have issue merging if it was client-only)

can remove if you want tho

Reply:
@astahmer I just felt like it's too react-ish 😁 related to what I brought up previously.